### PR TITLE
fix(Live-Annotation): remove the presenter side annotation info

### PR DESCRIPTION
**Pull request by perfect**
Feat(Live-Annotation): Remove the annotation info for share

https://confluence-eng-gpk2.cisco.com/conf/display/WTWC/New+Annotation+in+Webex+Web+Client+Design

## This pull request addresses

< DESCRIBE THE CONTEXT OF THE ISSUE >

## by making the following changes

< DESCRIBE YOUR CHANGES >

### Change Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/plugin-meetings/src/meeting/index.ts [hunk 1]
```diff
@@ -5395,9 +5394,6 @@ export default class Meeting extends StatelessWebexPlugin {
 
     this.audio = createMuteState(AUDIO, this, audioEnabled);
     this.video = createMuteState(VIDEO, this, videoEnabled);
-
-    this.annotationInfo = localTracks?.annotationInfo;
```
Comment (marcin-bazyl) [inline packages/@webex/plugin-meetings/src/meeting/index.ts:5399]: there is also the type declaration for `annotationInfo` around line 554

File: packages/@webex/plugin-meetings/src/meeting/index.ts [hunk 2]
```diff
@@ -150,7 +150,6 @@ export type LocalTracks = {
     audio?: LocalSystemAudioTrack;
     video?: LocalDisplayTrack;
   };
-  annotationInfo?: AnnotationInfo;
```
Comment (marcin-bazyl) [inline packages/@webex/plugin-meetings/src/meeting/index.ts:153]: also the import of `AnnotationInfo` can be removed now, right?

**Review by marcin-bazyl — COMMENTED** [review]
I think there are some unit tests that also need to be updated

File: packages/@webex/plugin-meetings/src/meeting/index.ts (hunk 2 above)
Comment (perfect) [reply packages/@webex/plugin-meetings/src/meeting/index.ts:153]: Remove it

File: packages/@webex/plugin-meetings/src/meeting/index.ts (hunk 1 above)
Comment (perfect) [reply packages/@webex/plugin-meetings/src/meeting/index.ts:5399]: remove it

**Review by marcin-bazyl — APPROVED** [review]

**Review by mickelr — APPROVED** [review]
LGTM

**Pull request merged**